src/engine/enginemaster.cpp

```diff
@@ -97,6 +100,13 @@ EngineMaster::EngineMaster(UserSettingsPointer pConfig,
 
     m_pMasterDelay = new EngineDelay(group, ConfigKey(group, "delay"));
     m_pHeadDelay = new EngineDelay(group, ConfigKey(group, "headDelay"));
+    m_pInputLatencyCompensationDelay = new EngineDelay(group,
+        ConfigKey(group, "inputLatency"));
+    m_pInputLatencyCompensationHeadphonesDelay = new EngineDelay(group,
+        ConfigKey(group, "headMixDelay"));
```

Comment:
Yes right, but the user does not have the benefit of the additional booth output. If we Mixxx booth into headphones, we can freely mix talkover to sidechain only, sidechain + master, sidechain + master + booth. Only in the later case he has the talkover mix on his headphone. Sidechain + master is a reasonable default.

I consider booth and the master part of the headphone, to be the same. Both is intended to monitor the output. Both schould normally not contain talkover. All use-cases for one apply also for the second.
Did you consider my drawing? Do you understand it?